netrd/distance/onion_divergence.py

```diff
@@ -0,0 +1,192 @@
+"""
+onion_divergence.py
+--------------------------
+
+The Jensen-Shannon divergence between the two graphs onion decomposition.
+
+Graph distance based on :
+https://www.nature.com/articles/srep31708
+https://journals.aps.org/prx/abstract/10.1103/PhysRevX.9.011023
+
+authors: Laurent Hébert-Dufresne and Guillaume St-Onge
+email: guillaume.st-onge.4@ulaval.ca
+Submitted as part of the 2019 NetSI Collabathon.
+
+"""
+
+import numpy as np
+import networkx as nx
+from .base import BaseDistance
+from functools import reduce
+
+class OnionDivergence(BaseDistance):
+    def dist(self, G1, G2, dist='lccm'):
+        """
+        Return the Jenson-Shannon divergence between two graphs onion spectra
+        of two graphs.
+
+        Note: The method assumes simple graphs.
+
+        Params
+        ------
+
+        G1, G2 (nx.Graph): two networkx graphs to be compared.
+        dist (str): type of distribution divergence to output. Choices are
```

Comment:
Could you add some documentation of the different dist types to the docstring? Without reading the code, I'm not sure what the difference between, say, `lccm` and `lccm_node` would be.